doc/source/operations/rocky-linux-9.rst

```diff
@@ -0,0 +1,517 @@
+
+4. Deprovision the controller:
+
+   .. code:: console
+
+      kayobe overcloud deprovision -l <hostname>
+
+5. Reprovision the controller:
+
+   .. code:: console
+
+      kayobe overcloud provision -l <hostname>
+
+6. Host configure:
+
+   .. code:: console
+
+      kayobe overcloud host configure -l <hostname> -kl <hostname>
+
+7. Service deploy on all controllers:
+
+   .. code:: console
+
+      kayobe overcloud service deploy -kl controllers
+
+8. If using OVN, check OVN northbound DB cluster state on all controllers to see if the new host has joined:
+
+   .. code:: console
+
+      kayobe overcloud host command run --command 'docker exec -it ovn_nb_db ovs-appctl -t /run/ovn/ovnnb_db.ctl cluster/status OVN_Northbound' --show-output -l controllers
+
+9. If using OVN, check OVN southbound DB cluster state on all controllers to see if the new host has joined:
+
+   .. code:: console
+
+      kayobe overcloud host command run --command 'docker exec -it ovn_sb_db ovs-appctl -t /run/ovn/ovnsb_db.ctl cluster/status OVN_Southbound' --show-output -l controllers
+
+10. Some MariaDB instability has been observed. The exact cause is unknown but
+    the simplest fix seems to be to run the Kayobe database recovery tool
+    between migrations.
+
+    .. code:: console
+
+       kayobe overcloud database recover
+
+After each controller has been migrated you may wish to perform some smoke testing, check for alerts and errors etc.
+
+Compute
+=======
+
+Compute nodes can be migrated to Rocky Linux 9 in batches.
+The possible batches depend on a number of things:
+
+* willingness for instance reboots and downtime
+* available spare hypervisor capacity
+* sizes of groups of compatible hypervisors
+
+Potential issues
+----------------
+
+Nothing yet!
+
+Full procedure for one batch of hosts
+-------------------------------------
+
+1. Disable the Nova compute service and drain it of VMs using live migration.
+   If any VMs fail to migrate, they may be cold migrated or powered off:
+
+   .. code-block:: console
+
+      kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/nova-compute-{disable,drain}.yml --limit <host>
+
+2. Deprovision the compute node:
+
+   .. code:: console
+
+      kayobe overcloud deprovision -l <hostname>
+
+3. Reprovision the compute node:
+
+   .. code:: console
+
+      kayobe overcloud provision -l <hostname>
+
+4. Host configure:
+
+   .. code:: console
+
+      kayobe overcloud host configure -l <hostname> -kl <hostname>
+
+5. Service deploy:
+
+   .. code:: console
+
+      kayobe overcloud service deploy -kl <hostname>
+
+If any VMs were powered off, they may now be powered back on.
+
+Wait for Prometheus alerts and errors in OpenSearch Dashboard to resolve, or
+address them.
+
+After updating controllers or network hosts, run any appropriate smoke tests.
+
+Once happy that the system has been restored to full health, move onto the next
+host or batch or hosts.
+
+Storage
```

Comment:
Could do with a note on controllers/compute section, that if you have Ceph on controllers (well, once customer has) or computes (probably more than one customer) - use the maintenance enter/exit steps from storage section ;-)